channels/es.m3u

```diff
@@ -427,10 +409,6 @@ https://stream.maestrat.tv/hls/stream.m3u8
 http://iptv.btpba1.es.network.do:8080/martv-web/video.m3u8
 #EXTINF:-1 tvg-id="mtmad24h.es" tvg-name="mtmad 24h" tvg-country="ES" tvg-language="" tvg-logo="" group-title="",mtmad 24h
 https://mdssplive-lh.akamaihd.net/i/mtmad_1@744785/index_2500_av-b.m3u8
-#EXTINF:-1 tvg-id="mtmad24h.es" tvg-name="mtmad 24h" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://i.imgur.com/I9QX555.png" group-title="",mtmad 24h
-https://mdssplive-lh.akamaihd.net/i/mtmad_1@744785/index_2500_av-p.m3u8
```

Comment:
https://mdssplive-lh.akamaihd.net/i/mtmad_1@744785/index.m3u8

Seems to be geoblocked.

Medianet ES channel.